Melinda

mel-d opened the pull request.

mel-d committed:
First commit

mel-d committed:
Finished with first five exercises

mel-d committed:
Fixed issues with number 5

mel-d committed:
Closest I've been able to get #6...

mel-d committed:
Working on #7

halfghaninne commented:
No.6 looks pretty good. Consider what else you might need to apply to each element of  `.festival` to give space between them (this may affect how you've set up spacing between your headers and the following elements in each div.month).

Then [these results](https://www.google.com/search?q=how+to+remove+link+styling&oq=how+to+remove+link+styling&aqs=chrome..69i57j0l4.6906j0j7&sourceid=chrome&es_sm=119&ie=UTF-8) should get you the rest of the way.

Taking a look at No.7 now!

halfghaninne commented:
For No.7 [this declaration](https://github.com/mel-d/css-toy__practice-problems/blob/master/07/global.css#L45) is certainly causing some issues.

mel-d commented:
Oh dang; I had just added that padding to screw around with the program and forgot to remove it before pushing--I fixed that but am still having spacing issues!

mel-d committed:
Working on spacing issues

mel-d commented:
I'll screw around with it again post-yoga and let you know if I still can't figure it out. Thanks!

mel-d committed:
Very little improved but I hate everything so I'm leaving #6 behind

mel-d committed:
Same for #7

mel-d committed:
I'm terrible at this.

mel-d committed:
Back to struggling with #5

mel-d commented:
Ok, so I'm trying to go back and make my exercises #5-8 look better after this morning's lecture, but I still feel like an idiot. I've gotten the first 1/3 or so of the screen spaced out right, but I still can't figure out where I have all the extra padding that's causing things to extend too far downwards.

sumeetjain commented:
Since you changed the HTML, it might no longer be valid code. Can you run your HTML through [this validator](https://validator.w3.org/#validate_by_input) and see if it returns any errors. It shouldn't, if it's valid code.

If it does return errors, it means the structure of your HTML is faulty somehow, which means the DOM will be incorrectly built (because the HTML for the DOM's structure is not correct). HTML doesn't give us handy errors like Ruby unless we run it through this validator (or another validator like it).

sumeetjain commented:
Once you fix the HTML to be valid, see if the display issues remain. If they do, re-submit the question and I'll take a closer look.

mel-d commented:
<img width="490" alt="screen shot 2016-03-18 at 11 20 43 am" src="https://cloud.githubusercontent.com/assets/16277895/13884619/c480e66c-ecfc-11e5-8b36-04c94232702e.png">
^ screen shot of this last question, just for future reference

mel-d committed:
Fixed html

mel-d commented:
<img width="450" alt="screen shot 2016-03-18 at 11 44 07 am" src="https://cloud.githubusercontent.com/assets/16277895/13885096/e771e656-ecfe-11e5-98dd-fb004f54b6b6.png">
Fixed my html--sorry about that. I don't know why I didn't notice those screwed up tags. I'm still having the same issue, though.

mel-d commented:
(specifically re: why there's so much space between the links and the more_info when I don't have any padding below links or above the more_info.)

sumeetjain commented:
But you do have padding on `p`, which is inside of `.more_info`. So the declarations being made on `p` are being inherited by all `<p>` elements on the page. If you want to overwrite the padding values being inherited by certain `<p>` tags, try `.more_info p` to single them out.

mel-d commented:
Those are only side paddings, though, aren't they? The only issues I'm having are with top and bottom.

sumeetjain commented:
Oh, oops–I looked at the wrong selector. Let me take a closer/better look.

mel-d commented:
As far as I can tell, the only thing that should be affecting anything below the white line is the h3 top-padding of 20px.

mel-d commented:
(and the 13px margin above links)

sumeetjain commented:
In `more-info`'s `<p>`, the first element is a line-break: `<br />` :) That's adding an empty line.

mel-d commented:
I didn't know I was allowed to edit that for this assignment--it fixed my problem, thanks!

mel-d committed:
#5 actually looks like it's supposed to--I don't know if I believe it

mel-d committed:
Finished fixing through exercise 7

mel-d committed:
#8 is looking acceptable-ish

mel-d committed:
Merge branch 'master' of github.com:omahacodeschool/css-toy__practice-problems

mel-d committed:
Number 9 completed

mel-d committed:
As close as I think I can get to number 10 without knowing the actual font?

mel-d committed:
Getting started on 11

mel-d committed:
Finished number 11

mel-d committed:
Html for exercise 12

mel-d committed:
Finished with 11--I think I'm overthinking this because floats still confuse the hell out of me

mel-d committed:
Recreated part of the NYT
